docs/docsource/operation/Orbit_UI/Bucket_Management/lifecycle_management/Lifecycle_Management-Expiration.rst

```diff
@@ -31,9 +31,9 @@ Establishing an Object Expiration Policy
       :align: center
 
    You may enter a distinct directory or subdirectory to which the rule applies.
-   Enter an expiration time span and a deletion time span.
-   These follow the bucket and enforce expiration and deletion.
-   You may also add a comment about this expiration rule.
+   Enter an expiration time span and a deletion time span.  These follow the
+   bucket and enforce expiration and deletion.  You may also add a comment about
```

Comment:
Double spaces?
```suggestion
   Enter an expiration time span and a deletion time span. These follow the
   bucket and enforce expiration and deletion. You may also add a comment about
```

Reply:
A little gift from Mr. Emacs.

Reply:
We don't double-space after periods.

Reply:
Right, I was removing one of the two spaces.

Reply:
One of these days I'm going to figure out how to use these tools. Either way it's a moot point, because the Orbit docs are all on the chopping block.